src/Modules/CalcOffence.lua

```diff
@@ -1192,6 +1192,37 @@ function calcs.offence(env, actor, activeSkill)
 		end
 	end
 
+	-- Skill uptime
+	do
+		local cooldown = output.Cooldown or 0
+		for _, durationType in pairs({ "Duration", "DurationSecondary", "AuraDuration", "reserveDuration" }) do
```

Comment:
This needs to exclude vaal auras as those have another factor that limits their uptime (souls) which if not calculated here may give users the wrong impression, and will lead to misunderstandings